data/curiousCases/VUEs.json

```diff
@@ -0,0 +1,122 @@
+[
+    {
+        "hugoGeneSymbol": "KIT",
+        "transcriptId": "ENST00000288135",
+        "genomicLocation": "Deletions flanking intron 10 - exon 11 boundary (4:55593576_55593606del)",
+        "defaultEffect": "splice",
+        "comment": "Changes splice donor site, cause inframe deletion in exon 11",
+        "context": "Actionable in GIST",
+        "pubmedIds": [15507676],
+        "referenceText": "Corless et al, 2004",
+        "revisedProteinEffects": [ 
+        {
+            "variant":"4:g.55593576_55593606del", 
+            "genomicLocation": "4,5,55593576,55593606,ATATAATATATATATAT,-",
```

Comment:
Sorry this is just an example string I put here, the real genomic location should be: `4,5,55593576,55593606,CCACAGAAACCCATGTATGAAGTACAGTGGA,-`